packages/iframe-coordinator/rollup.config.js

```diff
@@ -12,10 +12,14 @@ export default [
       file: 'dist/index.js',
```

Comment:
The changes in this file are to stop warnings in the build process. They do not change the output in any way.